### ISSUE TITLE
"could not decode input file" error in Xcode 10

### ISSUE THREAD

**Issue by rnikander** [uservoice/uservoice-ios-sdk#464]
I'm getting the following error trying to build with Xcode 10 beta. 

    CopyStringsFile /.../en.lproj/UserVoice.strings
        /.../UserVoiceSDK-3.2.12/UVResources/en.lproj/UserVoice.strings (in target: ...)
    cd /...
    builtin-copyStrings --validate --inputencoding utf-8 --outputencoding binary 
      --outdir /.../Library/Developer/Xcode/DerivedData/.../Build/Products/Debug-iphoneos/.../en.lproj 
      -- /.../UserVoiceSDK-3.2.12/UVResources/en.lproj/UserVoice.strings

    error: could not decode input file using specified encoding: Unicode (UTF-8)

**Comment by rnikander** [comment]
I think this was an Xcode bug. It's fixed in version 10 beta 5.

**Comment by mheins57** [comment]
I have Xcode 10 beta 5, and am still getting this error.

**Comment by mheins57** [comment]
Aha, I just figured out how to fix this, at least for my case.  In my case, I had 2 Localizable.strings files (one for English, one for German).  They were UTF-16 little endian, but Xcode 10 was trying to interpret them as UTF-8 and failing.  In Xcode I selected each file, and under Text Settings in the file inspector, I noticed that they were shown as "unspecified encoding".  Selecting Unicode (UTF-16) brought up a dialog with the choice to either convert or reinterpret, with this explanatory text: Choose “Reinterpret” if you believe the file has been opened with an incorrect encoding and you want to reopen it as “Unicode (UTF-16)”.  Choosing "Reinterpret" fixed the problem.
